geode-docs/developing/query_select/aggregates.html.md.erb

```diff
@@ -0,0 +1,340 @@
+
+``` pre
+SELECT pf.status, MIN(pf.ID), MAX(pf.ID), COUNT(pf.ID), AVG(pf.ID), SUM(pf.ID)
+FROM /portfolio pf
+GROUP BY pf.status
+```
+
+## <a id="min"></a>MIN
+
+The `MIN` keyword returns the minimum or smallest value from the selected expression. The expression itself must always evaluate to `java.lang.Comparable`. The `MIN` statement returns the actual type of the selected element as its result. 
+
+The following are example `MIN` queries that return region entries (the entries implement the `java.lang.Comparable` interface).
+
+``` pre
+SELECT MIN(pf)
+FROM /exampleRegion pf
+```
+
+``` pre
+SELECT MIN(pf)
+FROM /exampleRegion
+pf WHERE pf.ID > 0
+```
+
+``` pre
+SELECT MIN(pf)
+FROM /exampleRegion pf
+WHERE pf.ID > 10 LIMIT 50
+```
+
+``` pre
+SELECT MIN(pf) 
+FROM /exampleRegion pf 
+WHERE pf.ID > 0 AND pf.status LIKE 'act%'
+```
+
+The following `MIN` query returns the lowest entry ID that matches the query's selection criteria.
+
+``` pre
+SELECT MIN(pf.ID)
+FROM /exampleRegion pf, pf.positions.values pos
+WHERE pf.ID > 0 AND pos.secId = 'IBM'
+```
+
+The following `MIN` query returns the lowest positive ID grouped by status.
+
+``` pre
+SELECT pf.status, MIN(pf.ID)
+FROM /exampleRegion pf
+WHERE pf.ID > 0
+GROUP BY pf.status
+```
+
+## <a id="max"></a>MAX
+
+The `MAX` keyword returns the maximum or highest value from the selected expression. The expression itself must always evaluate to `java.lang.Comparable`. The `MAX` statement returns the type of the selected element as its result. 
+
+The following are example `MAX` queries that return region entries (the entries implement the `java.lang.Comparable` interface).
+
+``` pre
+SELECT MAX(pf)
+FROM /exampleRegion pf
+```
+
+``` pre
+SELECT MAX(pf)
+FROM /exampleRegion
+pf WHERE pf.ID > 0
+```
+
+``` pre
+SELECT MAX(pf)
+FROM /exampleRegion pf
+WHERE pf.ID > 10 LIMIT 50
+```
+
+``` pre
+SELECT MAX(pf) 
+FROM /exampleRegion pf 
+WHERE pf.ID > 0 AND pf.status LIKE 'act%'
+```
+
+The following `MAX` query returns the highest entry ID that matches the query's selection criteria.
+
+``` pre
+SELECT MAX(pf.ID)
+FROM /exampleRegion pf, pf.positions.values pos
+WHERE pf.ID > 0 AND pos.secId = 'IBM'
+```
+
+The following `MAX` query returns the highest positive IDs grouped by status.
+
+``` pre
+SELECT pf.status, MAX(pf.ID)
+FROM /exampleRegion pf
+WHERE pf.ID > 0
+GROUP BY pf.status
+```
+
+## <a id="count"></a>COUNT
+
+The `COUNT` keyword returns the number of results that match the query selection conditions specified in the `WHERE` clause. Using `COUNT` allows you to determine the size of a results set.
+The `COUNT` statement always returns a `java.lang.Integer` or `java.lang.Long` as the result (depending on how big the value is); you should take this into consideration when executing the query: if an overflow occurs while computing the `COUNT` function because the value is higher than `Long.MAX_VALUE` (2<sup>63</sup> - 1), the result will be incorrect.
```

Comment:
Is this statement about returning integer true still?

Reply:
Yes, it' still true. We internally downcast the value before returning it to the user, meaning that they can get a ` java.lang.Integer` or a `java.lang.Long` as the result, depending on "how big" the actual value is.